feat(curriculum): Add English tasks B1 Block 6

miyaliu666 opened the pull request:
Checklist:

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

I've adjusted a few audio timings. Some audios are missing one or two words at the end, but they play normally when clicked again.

<!-- Feel free to add any additional description of changes below this line -->

miyaliu666 committed:
created tasks for dialogue 1

miyaliu666 committed:
created more tasks for dialogue 1

miyaliu666 committed:
finished dialogue 1

miyaliu666 committed:
added # in video solutions

miyaliu666 committed:
created tasks for dialogue 2

miyaliu666 committed:
created tasks for dialogue 2

miyaliu666 committed:
update meta

miyaliu666 committed:
finished creating tasks for dialogue 2

miyaliu666 committed:
updated meta

miyaliu666 committed:
created tasks for dialogue 3

miyaliu666 committed:
remove extra parts

miyaliu666 committed:
created more tasks for dialogue 3

miyaliu666 committed:
finished creating tasks

miyaliu666 committed:
fixed errors

naomi-lgbt commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/675fba2a32b4aa625e2f7940.md:
```diff
@@ -0,0 +1,38 @@
+
+## --text--
+
+What's the right reply when your team member asks `how often should we review our security policies`?
```
```suggestion
What's the right reply when your team member asks `How often should we review our security policies`?
```

naomi-lgbt reviewed (changes requested):
Just a few quick nits.

anastasiiauk commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/676147de99ef019cee551581.md:
```diff
@@ -0,0 +1,70 @@
+
+# --questions--
+
+## --text--
+
+`Dear Team,`
+
+`As the summit is near and many of you will be traveling for work, it’s important to stay safe online. Man-in-the-Middle attacks (MitM) often happen on public Wi-Fi or unsecured networks, especially during peak travel times. These attacks can steal your data, including corporate emails and sensitive information.`
+
+`To stay protected, always use a VPN (Virtual Private Network) when accessing sensitive information or company systems. Avoid using public Wi-Fi for important tasks. Check the updated security tips and job aids we’ve shared. They include simple steps to keep your data safe while traveling.`
+
+`We’ve scheduled a security review meeting to share more advice and discuss security protocols. If you have any questions or need help setting up your VPN, contact the security team before your trip.`
+
+`Stay safe,`
+
+`Your Security Team`
```
Added introduction and changed apostrophes.

anastasiiauk commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/676147de99ef019cee551581.md:
```diff
@@ -0,0 +1,70 @@
+
+### --feedback--
+
+The email does not mention delays as a reason for staying secure online. Focus on the risk of data being stolen.
```
```suggestion
The email does not mention delays as a reason for staying secure online.
```
This feedback gives away the answer.

anastasiiauk commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/676147de99ef019cee551581.md:
```diff
@@ -0,0 +1,70 @@
+
+### --feedback--
+
+The email does not mention anything about connecting to a summit network. The focus is on protecting sensitive information.
```
```suggestion
The email does not mention anything about connecting to a summit network.
```
This feedback gives away the answer.

anastasiiauk commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/676147de99ef019cee551581.md:
```diff
@@ -0,0 +1,70 @@
+
+# --explanation--
+
+To find the correct answer, focus on the email’s warning about Man-in-the-Middle attacks.
```
```suggestion
To find the correct answer, focus on the email's warning about Man-in-the-Middle attacks.
```

anastasiiauk commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/6761493d2db43ba139340975.md:
```diff
@@ -0,0 +1,70 @@
+
+# --questions--
+
+## --text--
+
+`Dear Team,`
+
+`As the summit is near and many of you will be traveling for work, it’s important to stay safe online. Man-in-the-Middle attacks (MitM) often happen on public Wi-Fi or unsecured networks, especially during peak travel times. These attacks can steal your data, including corporate emails and sensitive information.`
+
+`To stay protected, always use a VPN (Virtual Private Network) when accessing sensitive information or company systems. Avoid using public Wi-Fi for important tasks. Check the updated security tips and job aids we’ve shared. They include simple steps to keep your data safe while traveling.`
+
+`We’ve scheduled a security review meeting to share more advice and discuss security protocols. If you have any questions or need help setting up your VPN, contact the security team before your trip.`
+
+`Stay safe,`
+
+`Your Security Team`
```
```suggestion
## --text--

After discussing the risks with Brian, Anna decided to inform the team about preventing MitM attacks, especially during work-related travel:

`Dear Team,`

`As the summit is near and many of you will be traveling for work, it's important to stay safe online. Man-in-the-Middle attacks (MitM) often happen on public Wi-Fi or unsecured networks, especially during peak travel times. These attacks can steal your data, including corporate emails and sensitive information.`

`To stay protected, always use a VPN (Virtual Private Network) when accessing sensitive information or company systems. Avoid using public Wi-Fi for important tasks. Check the updated security tips and job aids we've shared. They include simple steps to keep your data safe while traveling.`

`We've scheduled a security review meeting to share more advice and discuss security protocols. If you have any questions or need help setting up your VPN, contact the security team before your trip.`

`Stay safe,`
`Your Security Team`
```

anastasiiauk commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/6761493d2db43ba139340975.md:
```diff
@@ -0,0 +1,70 @@
+
+# --explanation--
+
+To find the correct answer, focus on the email’s advice about public Wi-Fi.
```
```suggestion
To find the correct answer, focus on the email's advice about public Wi-Fi.
```

anastasiiauk commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/67614992d9e494a2b69f89bb.md:
```diff
@@ -0,0 +1,70 @@
+
+# --questions--
+
+## --text--
+
+`Dear Team,`
+
+`As the summit is near and many of you will be traveling for work, it’s important to stay safe online. Man-in-the-Middle attacks (MitM) often happen on public Wi-Fi or unsecured networks, especially during peak travel times. These attacks can steal your data, including corporate emails and sensitive information.`
+
+`To stay protected, always use a VPN (Virtual Private Network) when accessing sensitive information or company systems. Avoid using public Wi-Fi for important tasks. Check the updated security tips and job aids we’ve shared. They include simple steps to keep your data safe while traveling.`
+
+`We’ve scheduled a security review meeting to share more advice and discuss security protocols. If you have any questions or need help setting up your VPN, contact the security team before your trip.`
+
+`Stay safe,`
+
+`Your Security Team`
```
```suggestion
## --text--

After discussing the risks with Brian, Anna decided to inform the team about preventing MitM attacks, especially during work-related travel:

`Dear Team,`

`As the summit is near and many of you will be traveling for work, it's important to stay safe online. Man-in-the-Middle attacks (MitM) often happen on public Wi-Fi or unsecured networks, especially during peak travel times. These attacks can steal your data, including corporate emails and sensitive information.`

`To stay protected, always use a VPN (Virtual Private Network) when accessing sensitive information or company systems. Avoid using public Wi-Fi for important tasks. Check the updated security tips and job aids we've shared. They include simple steps to keep your data safe while traveling.`

`We've scheduled a security review meeting to share more advice and discuss security protocols. If you have any questions or need help setting up your VPN, contact the security team before your trip.`

`Stay safe,`
`Your Security Team`
```

anastasiiauk commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/67614992d9e494a2b69f89bb.md:
```diff
@@ -0,0 +1,70 @@
+
+### --feedback--
+
+The job aid provides guidance, but the email specifically mentions a VPN as a key tool for protection.
```
```suggestion
The job aid provides guidance, but the email mentions a specific key tool for protection.
```
This feedback gives away the answer.

anastasiiauk commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/67614a0187bd80a49d438b1f.md:
```diff
@@ -0,0 +1,70 @@
+
+# --questions--
+
+## --text--
+
+`Dear Team,`
+
+`As the summit is near and many of you will be traveling for work, it’s important to stay safe online. Man-in-the-Middle attacks (MitM) often happen on public Wi-Fi or unsecured networks, especially during peak travel times. These attacks can steal your data, including corporate emails and sensitive information.`
+
+`To stay protected, always use a VPN (Virtual Private Network) when accessing sensitive information or company systems. Avoid using public Wi-Fi for important tasks. Check the updated security tips and job aids we’ve shared. They include simple steps to keep your data safe while traveling.`
+
+`We’ve scheduled a security review meeting to share more advice and discuss security protocols. If you have any questions or need help setting up your VPN, contact the security team before your trip.`
+
+`Stay safe,`
+
+`Your Security Team`
```
```suggestion
## --text--

After discussing the risks with Brian, Anna decided to inform the team about preventing MitM attacks, especially during work-related travel:

`Dear Team,`

`As the summit is near and many of you will be traveling for work, it's important to stay safe online. Man-in-the-Middle attacks (MitM) often happen on public Wi-Fi or unsecured networks, especially during peak travel times. These attacks can steal your data, including corporate emails and sensitive information.`

`To stay protected, always use a VPN (Virtual Private Network) when accessing sensitive information or company systems. Avoid using public Wi-Fi for important tasks. Check the updated security tips and job aids we've shared. They include simple steps to keep your data safe while traveling.`

`We've scheduled a security review meeting to share more advice and discuss security protocols. If you have any questions or need help setting up your VPN, contact the security team before your trip.`

`Stay safe,`
`Your Security Team`
```

anastasiiauk commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/67614a0187bd80a49d438b1f.md:
```diff
@@ -0,0 +1,70 @@
+
+# --explanation--
+
+To find the correct answer, focus on the email’s mention of updated security tips.
```
```suggestion
To find the correct answer, focus on the email's mention of updated security tips.
```

anastasiiauk commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/67614a6e5d4df6a6744127b7.md:
```diff
@@ -0,0 +1,70 @@
+
+### --feedback--
+
+While reviewing the tips is helpful, the email emphasizes contacting the security team for VPN assistance as the most important step.
```
```suggestion
While reviewing the tips is helpful, the email emphasizes something else as the most important step.
```
This feedback gives away the answer.

anastasiiauk commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/67614a6e5d4df6a6744127b7.md:
```diff
@@ -0,0 +1,70 @@
+
+# --explanation--
+
+To find the correct answer, focus on the email’s advice for preparing before traveling.
```
```suggestion
To find the correct answer, focus on the email's advice for preparing before traveling.
```

anastasiiauk reviewed (changes requested):
I loved your tasks so much!! They're structured and follow our guidelines perfectly. 
Most of my comments address changing explicit feedbacks and changing characters.

DanielRosa74 committed:
fix(curriculum): fix new character positioning issue and apply Naomi's suggestions

DanielRosa74 committed:
fix(curriculum): fix task 48 challenge type

DanielRosa74 committed:
fix(curriculum): fix audio sync and quotes

DanielRosa74 commented:
I made changes to the start times of audio during my review. I also added Naomi's suggestions as per conversation with @miyaliu666 and changed single curly quotes to straight quotes to match other cases. Apart from that, block looks good to me. I'm also resolving conversations that are solved by the changes I implemented.

DanielRosa74 commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/6752e83a806f520923bd28fa.md:
```diff
@@ -0,0 +1,125 @@
+
+### --feedback--
+
+Brian clearly agrees with Jessica’s plan, calling it a "Good idea," so this is incorrect.
```
Partially solved. Apostrophe is now straight instead of curly.

DanielRosa74 commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/6759a3d84d89c4474d6a06b6.md:
```diff
@@ -0,0 +1,102 @@
+
+### --feedback--
+
+Mark does not mention the system’s speed; the issue is about logging in.
```
Partially solved. Apostrophe is now a straight quote instead of curly.

DanielRosa74 commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/675e7cc999813e83379b139c.md:
```diff
@@ -0,0 +1,64 @@
+
+`The first step is to analyze the server logs. If we can’t find the cause in the logs, the next step is to replicate the issue on a staging server. Testing on a staging server is safe because it allows us to try different solutions without affecting the live system.`
+
+`When login problems continue, support tickets will start piling up, won’t they? Solving these issues quickly is not just about fixing the system — it’s also about maintaining user trust and satisfaction, isn’t it? Let’s work together, find the root cause, and resolve this problem efficiently.`
```
Partially solved. Apostrophes are now single straight quotes instead of curly.

DanielRosa74 commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/675e8013d40d0d860143c14f.md:
```diff
@@ -0,0 +1,64 @@
+
+`When login problems continue, support tickets will start piling up, won’t they? Solving these issues quickly is not just about fixing the system — it’s also about maintaining user trust and satisfaction, isn’t it? Let’s work together, find the root cause, and resolve this problem efficiently.`
+
+What happens if tokens aren’t renewed on time?
```
Partially solved. Apostrophes are now single straight quotes instead of curly.

DanielRosa74 commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/675e8133c874a588f947de57.md:
```diff
@@ -0,0 +1,64 @@
+
+`When login problems continue, support tickets will start piling up, won’t they? Solving these issues quickly is not just about fixing the system — it’s also about maintaining user trust and satisfaction, isn’t it? Let’s work together, find the root cause, and resolve this problem efficiently.`
+
+What does Mark suggest if the server logs don’t show the cause of the problem?
```
Partially solved. Apostrophes are now single straight quotes instead of curly.

miyaliu666 commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/674f06b4b4d6eb4f6139ad61.md:
```diff
@@ -0,0 +1,52 @@
+
+## --text--
+
+What might be involved in website tests?
```
I considered your suggestion, but since "website tests" is one of the vocabulary items we want to teach in this dialogue, I think it's fine to test understanding of this phrase after teaching the individual words in task 1.

miyaliu666 commented on curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/6752f4680f2d251372d4ae64.md:
```diff
@@ -0,0 +1,70 @@
+
+## --text--
+
+`After discussing with Brian, Jessica wrote this scheduled maintenance notification:`
```
@nieldakarla do we want to include this into the guideline?

miyaliu666 replied:
> It looks like an introduction to the reading text, so there shouldn't be backticks.

@nieldakarla do we want to include this into the guideline?

miyaliu666 committed:
Apply Anastasiia's review suggestions

Co-authored-by: Anastasiia <109740924+anastasiiauk@users.noreply.github.com>

DanielRosa74 committed:
Update curriculum/challenges/english/24-b1-english-for-developers/learn-future-continuous-while-describing-actions/674f0f8e84b4c76c0f6ae6d3.md

Co-authored-by: Anastasiia <109740924+anastasiiauk@users.noreply.github.com>

DanielRosa74 commented:
The team is still working on implementing @anastasiiauk's suggestions. Please do not merge this yet.

DanielRosa74 committed:
fix(curriculum): fix lint issues

DanielRosa74 committed:
feat(curriculum):implement anastasiiauk's changes - 1

feat(curriculum):implement anastasiiauk's changes - 2

feat(curriculum):implement anastasiiauk's changes - 3

feat(curriculum):implement anastasiiauk's changes - 4

feat(curriculum):implement anastasiiauk's changes - 5

feat(curriculum):implement anastasiiauk's changes - 6

feat(curriculum):implement anastasiiauk's changes - 7

feat(curriculum):implement anastasiiauk's changes - 8

feat(curriculum):implement anastasiiauk's changes - 9

feat(curriculum):implement anastasiiauk's changes - 10
